src/gui/components/tabs/tabs.styles.js

```diff
@@ -0,0 +1,8 @@
+import { css } from 'glamor'
+
+const layout = css({
+  position: 'relative',
+  border: '1px solid #ddd',
```

Comment:
ah... I'm not agree with this one.
I don't want to have a border of '1.333px' if I pass my font from '10px' to '13px'.
I want to have a constant size of '1px' here on the border.